yt_dlp/extractor/veoh.py

```diff
@@ -51,30 +60,36 @@ class VeohIE(InfoExtractor):
     }, {
         'url': 'http://www.veoh.com/watch/e152215AJxZktGS',
         'only_matching': True,
-    }]
-
-    def _extract_video(self, source):
-        return {
-            'id': source.get('videoId'),
-            'title': source.get('title'),
-            'description': source.get('description'),
-            'thumbnail': source.get('highResImage') or source.get('medResImage'),
-            'uploader': source.get('username'),
-            'duration': int_or_none(source.get('length')),
-            'view_count': int_or_none(source.get('views')),
-            'age_limit': 18 if source.get('isMature') == 'true' or source.get('isSexy') == 'true' else 0,
-            'formats': self._extract_formats(source),
+    }, {
+        'url': 'https://www.veoh.com/videos/v16374379WA437rMH',
+        'md5': 'cceb73f3909063d64f4b93d4defca1b3',
+        'info_dict': {
+            'id': 'v16374379WA437rMH',
+            'ext': 'mp4',
+            'title': 'Phantasmagoria 2, pt. 1-3',
+            'description': 'Phantasmagoria: a Puzzle of Flesh',
+            'thumbnail': 're:https://fcache\\.veoh\\.com/file/f/th16374379\\.jpg(\\?.*)?',
+            'uploader': 'davidspackage',
+            'duration': 968,
+            'view_count': int,
+            'average_rating': int,
+            'comment_count': int,
+            'age_limit': 18,
+            'categories': ['technology_and_gaming', 'gaming'],
+            'tags': ['puzzle', 'of', 'flesh'],
         }
+    }]
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
-        video = self._download_json(
+        json = self._download_json(
```

Comment:
confusing variable name

Reply:
Is the problem that it's been parsed and therefore isn't JSON _anymore_, that it doesn't specify what was _in_ the JSON, that it implies that `video` isn't also parsed JSON although it is, or something else?

Is this acceptable? (Would appreciate your reasoning regardless; I looked at usage in other extractors and was not able to determine a consistent style.)

Reply:
It's because `json` is also the name of a standard library

Reply:
Ahhh. Thanks.